app/uk/gov/hmrc/gform/fileupload/FileUploadModule.scala

```diff
@@ -20,7 +20,7 @@ import uk.gov.hmrc.gform.config.ConfigModule
 import uk.gov.hmrc.gform.time.{ TimeModule, TimeProvider }
 import uk.gov.hmrc.gform.wshttp.WSHttpModule
 
-class FileUploadModule(configModule: ConfigModule, wSHttpModule: WSHttpModule, timeModule: TimeModule) {
+class FileUploadModule(configModule: ConfigModule, val wSHttpModule: WSHttpModule, timeModule: TimeModule) {
```

Comment:
Let's remove this `val` keyword. We don't want to expose sub-modules from module. Module suppose to be responsible only for creating instances.